scripts/e2e-tests.sh

```diff
@@ -261,30 +258,26 @@ function setup_test_cluster() {
   set -o errexit
   set -o pipefail
 
-  # Set the required variables if necessary.
-  if [[ -z ${K8S_USER_OVERRIDE} ]]; then
-    export K8S_USER_OVERRIDE=$(gcloud config get-value core/account)
-  fi
+  local k8s_user=$(gcloud config get-value core/account)
+  local k8s_cluster=$(kubectl config current-context)
+  local clusterRoleBinding=$(kubectl get clusterrolebinding cluster-admin-binding)
 
-  if [[ -z ${K8S_CLUSTER_OVERRIDE} ]]; then
+  if [[ -z ${clusterRoleBinding} ]]; then
```

Comment:
Here I add a guard to check if the binding of `cluster-admin-binding` already exists to solve the problem in https://github.com/knative/test-infra/pull/481#discussion_r257326543. Is this OK? @adrcunha

Reply:
Yes, thanks, that's the place. However, you don't need to store the result. You can do just

```
  # Acquire cluster admin role in case user doesn't have it
  if [[ -z "$(kubectl get clusterrolebinding cluster-admin-binding 2> /dev/null)" ]]; then
```

this will also avoid a spurious error message when running with kubetest.

Reply:
Done